README.md

```diff
@@ -108,16 +108,68 @@ skew may result in `system time has changed` messages in VM logs.
 #### Network
 
 The guest agent uses network interface metadata to manage the network
-interfaces in the guest by performing the following tasks:
-
-*   Enabled all associated network interfaces on boot.
+interfaces by performing the following tasks:
+
+*   Enable all associated network interfaces on boot.
+    *   Detect the current active network manager service that is managing the
+        primary NIC.
+    *   Rollback and delete any files/configurations left behind by all other
```

Comment:
outline that we rollback guest-agent managed files - not user's.

Reply:
Done.